lms/djangoapps/courseware/module_render.py

```diff
@@ -450,6 +451,17 @@ def handle_grade_event(block, event_type, event):  # pylint: disable=unused-argu
             descriptor.location,
         )
 
+        # Send a signal out to any listeners who are waiting for score change
+        # events.
+        SCORE_CHANGED.send(
+            sender=None,
+            points_possible=event['max_value'],
+            points_earned=event['value'],
+            user=user,
+            course=course_id,
+            usage=descriptor.location
+        )
```

Comment:
Shouldn't this only be sent when a score actually changes? If I get something wrong 10 times in a row, there shouldn't be signals being sent for those, right? Or do we want to capture that case as well?